From 083414497af5e733dcdb8b7d3b638b5514cc9242 Mon Sep 17 00:00:00 2001
From: Stephano Cetola <stephano@cetola.net>
Date: Wed, 8 Jul 2026 17:30:00 -0800
Subject: [PATCH 07/14] mnt-reform-defconfig: enable NVMe, GPIO SPI master
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Enable CONFIG_BLK_DEV_NVME=y: the PCIe host driver was already
present but the NVMe block driver was missing, so NVMe SSDs attached
via PCIe were never enumerated.

Enable CONFIG_DRIVER_SPI_GPIO=y: GPIO bitbang SPI master required
for the RP2040 system controller connection on the MNT CM4 carrier.
The spi-gpio DTS node (separate patch) depends on this driver.

Signed-off-by: Stephano Cetola <stephano@cetola.net>
---
 mnt-reform-defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mnt-reform-defconfig b/mnt-reform-defconfig
index 045856e980..bc9d0626c8 100644
--- a/mnt-reform-defconfig
+++ b/mnt-reform-defconfig
@@ -685,7 +685,7 @@ CONFIG_NET_USB_RTL8152=y
 #
 CONFIG_SPI=y
 # CONFIG_SPI_MEM is not set
-# CONFIG_DRIVER_SPI_GPIO is not set
+CONFIG_DRIVER_SPI_GPIO=y
 # CONFIG_DRIVER_SPI_LITEX_SPIFLASH is not set
 CONFIG_SPI_ROCKCHIP=y
 # end of SPI drivers
@@ -1071,7 +1071,7 @@ CONFIG_ROCKCHIP_PM_DOMAINS=y
 #
 # NVME Support
 #
-# CONFIG_BLK_DEV_NVME is not set
+CONFIG_BLK_DEV_NVME=y
 # end of NVME Support

 CONFIG_REBOOT_MODE=y
--
2.43.0

